qiskit/schemas/qobj_schema.json

```diff
@@ -273,10 +273,12 @@
                                 "parametric_pulse"
```

Comment:
When we settle on a schema implementation this should be mirrored to https://github.com/Qiskit/ibm-quantum-schemas.